tikv-client/src/main/java/com/pingcap/tikv/meta/TiDAGRequest.java

```diff
@@ -198,6 +206,7 @@ public long mask(long flags) {
 
   private TiTableInfo tableInfo;
   private List<TiPartitionDef> prunedParts;
+  private boolean useTiFlash;
```

Comment:
is it possible that we can make it final?

Reply:
We have to set the value from TiStrategy.